CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 * Improve performance of change notifications on nested collections somewhat ([PR #7402](https://github.com/realm/realm-core/pull/7402)).
 * Improve performance of aggregate operations on Dictionaries of objects, particularly when the dictionaries are empty ([PR #7418](https://github.com/realm/realm-core/pull/7418))
 * Added Resumption delay configuration to SyncClientTimeouts. ([PR #7441](https://github.com/realm/realm-core/pull/7441))
+* Update C-API log callback signature to include the log category. ([RCORE-2027](https://jira.mongodb.org/browse/RCORE-2027)
```

Comment:
I'm not sure this is the right approach. Don't we need a new release? @jedelbo

Reply:
Ups.. should update the changelog, as this pr superceedes #7477 .. also, I suppose we still use refs to github PRs in the changelog.. this Jira stuff is inherited